README.md

```diff
@@ -7,6 +7,9 @@ Test Pilot
 
 [![Coverage status](https://img.shields.io/coveralls/mozilla/testpilot/master.svg)](https://coveralls.io/r/mozilla/testpilot)
 
+Docker build: [![Circle CI](https://circleci.com/gh/mozilla/testpilot/tree/master.svg?style=svg&circle-token=88ea3e1a6d9b7558092b75358c6ab9251739b9b5)](https://circleci.com/gh/mozilla/testpilot/tree/master)
```

Comment:
Would be cool if we could make this URL public -- I don't know if that's a circle ci setting or what.  r+ on this patch though, thanks.